docs/blog/_posts/2020-04-29-24th-dotty-milestone-release.md

```diff
@@ -0,0 +1,155 @@
+   |             Required: Int
+```
+
+Now, the above error message also contains the following sentence:
+
+```
+   |             Maybe you are missing an else part for the conditional?
+```
+
+We hope this change will make the language more intuitive for new users.
+
+# Inline overrides
+Inline overrides are now supported. For example, consider the following code:
+
+```scala
+abstract class A:
+  def f(x: Int) = s"Foo $x"
+
+class B extends A:
+  inline override def f(x: Int) = s"Bar $x"
+
+@main def Test =
+  val b = B()
+  println(b.f(22))
+  val a: A = b
+  println(a.f(22))
+```
+
+The output of the above program is:
+
+```
+Bar 22
+Bar 22
+```
+
+This new change, however, comes with rather intricated rules – if you are interested to learn about them in details, see [documentation](https://dotty.epfl.ch/docs/reference/metaprogramming/inline.html#rules-for-overriding) on inlines and the PR #[8543](https://github.com/lampepfl/dotty/pull/8543/files) which introduced the change.
+
+# Let us know what you think!
+
+If you have questions or any sort of feedback, feel free to send us a message on our
+[Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
+[open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
+
+## Contributing
+
+Thank you to all the contributors who made this release possible 🎉
+
+According to `git shortlog -sn --no-merges 0.23.0-RC1..0.24.0-RC1` these are:
+
+```
+   136  Martin Odersky
+    74  Nicolas Stucki
+    37  Guillaume Martres
+    33  Robert Stoll
+    22  Liu Fengyun
+    19  Anatolii Kmetiuk
+    16  Arnaud ESTEVE
+    15  Olivier Blanvillain
+    10  Arnaud Esteve
+     9  Martijn Hoekstra
+     6  Anatolii
+     4  Som Snytt
+     4  bishabosha
+     4  Aleksander Boruch-Gruszecki
+     3  Miles Sabin
+     2  odersky
+     2  Fengyun Liu
+     2  Julien Richard-Foy
+     1  Ara Adkins
+     1  Maxime Kjaer
+     1  Philippus
+     1  Rike-Benjamin Schuppner
+     1  Julien Jean Paul Sirocchi
+     1  Dani Rey
+     1  Sébastien Doeraene
+     1  aesteve
+     1  Dale Wijnand
+     1  fhackett
+     1  gzoller
+     1  Michael Pilquist
+```
+
+If you want to get your hands dirty and contribute to Dotty, now is a good time to get involved!
+Head to our [Getting Started page for new contributors](https://dotty.epfl.ch/docs/contributing/getting-started.html),
+and have a look at some of the [good first issues](https://github.com/lampepfl/dotty/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%3Anovice).
+They make perfect entry points into hacking on the compiler.
+
+We are looking forward to having you join the team of contributors.
+
+## Library authors: Join our community build
+
+Dotty now has a set of widely-used community libraries that are built against every nightly Dotty
+snapshot. Currently, this includes shapeless, ScalaPB, algebra, scalatest, scopt and squants.
+Join our [community build](https://github.com/lampepfl/dotty-community-build)
```

Comment:
This link leads to the archived repository, one has to click https://github.com/lampepfl/dotty/tree/master/community-build again.